<launch>
	<!-- declare arg to be passed in -->
	<arg name="map" default="tmap_II"/> 
	<arg name="node_by_node" default="false"/>
	<arg name="machine" default="localhost" />
	<arg name="user" default="" />

	<machine name="$(arg machine)" address="$(arg machine)" env-loader="/opt/strands/strands_catkin_ws/devel/env.sh" user="$(arg user)" default="true"/>

	<node pkg="topological_navigation" name="topological_localisation" type="localisation.py" output="screen" args="$(arg map)"/>
	<node pkg="topological_navigation" name="topological_navigation" type="navigation.py" output="screen" args="$(arg map) $(arg node_by_node)"/>
	<!--<node pkg="topological_utils" type="visualise_map.py" name="visualise_map" args="$(arg map)"/>-->
	<node pkg="topological_map_manager" type="visualise_map.py" name="visualise_map" args="$(arg map)"/>
</launch>
